daemon/pool.go

```diff
@@ -131,10 +132,14 @@ func (dp *DriverPool) Start(ctx context.Context) error {
 	// But an alternative is to re-implement a root Context, which is even worse.
 	dp.poolCtx, dp.stop = context.WithCancel(context.Background())
 
+	// Channel acts like a deferred signal, so it should have a buffer.
```

Comment:
s/should/must/

Please also consider mentioning _who_ reads and writes this channel (the following may be wrong): "This channel is read by the pool manager goroutine as a signal to rescale. Senders write to this channel whenever an event occurs that may affect pool size, such as a driver being returned. The channel must have a buffer of at least 1 and sends to the channel should not block."

(Is there actually more than one place where this write happens? If so we might think about putting a method on it to ensure we always do the nonblocking send)